### ISSUE TITLE
yolo11n_seg_pnnx.py 与文档中的不一致

### ISSUE THREAD

**Issue by yangl720** [nihui/ncnn-android-yolo11#11]
这是文档中修改前的：
v_202 = v_201.view(1, 32, 6400)
v_208 = v_207.view(1, 32, 1600)
v_214 = v_213.view(1, 32, 400)
v_215 = torch.cat((v_202, v_208, v_214), dim=2)
# ...
v_261 = v_230.view(1, 144, 6400)
v_262 = v_245.view(1, 144, 1600)
v_263 = v_260.view(1, 144, 400)
v_264 = torch.cat((v_261, v_262, v_263), dim=2)
# ...
v_285 = (v_284, v_196, )
return v_285

这是自己执行pnnx  yolo11n-seg.torchscript后生成的对应位置：
v_202 = self.model_23_cv4_0_1_conv(v_201)
        v_203 = self.pnnx_unique_64(v_202)
        v_204 = self.model_23_cv4_0_2(v_203)
        v_205 = [v_199.size(0), 32, -1]
        v_206 = v_204.view(*v_205)
        v_207 = self.model_23_cv4_1_0_conv(v_163)
        v_208 = self.pnnx_unique_65(v_207)
        v_209 = self.model_23_cv4_1_1_conv(v_208)
        v_210 = self.pnnx_unique_66(v_209)
        v_211 = self.model_23_cv4_1_2(v_210)
        v_212 = v_211.view(*v_205)
        v_213 = self.model_23_cv4_2_0_conv(v_190)
        v_214 = self.pnnx_unique_67(v_213)
        v_215 = self.model_23_cv4_2_1_conv(v_214)
        v_216 = self.pnnx_unique_68(v_215)
        v_217 = self.model_23_cv4_2_2(v_216)
        v_218 = v_217.view(*v_205)
        v_219 = torch.cat((v_206, v_212, v_218), dim=2)
        v_220 = self.model_23_cv2_0_0_conv(v_148)
        v_221 = self.pnnx_unique_69(v_220)
        v_222 = self.model_23_cv2_0_1_conv(v_221)
        v_223 = self.pnnx_unique_70(v_222)
        v_224 = self.model_23_cv2_0_2(v_223)
        v_225 = self.model_23_cv3_0_0_0_conv(v_148)
        v_226 = self.pnnx_unique_71(v_225)
        v_227 = self.model_23_cv3_0_0_1_conv(v_226)
        v_228 = self.pnnx_unique_72(v_227)
        v_229 = self.model_23_cv3_0_1_0_conv(v_228)
        v_230 = self.pnnx_unique_73(v_229)
        v_231 = self.model_23_cv3_0_1_1_conv(v_230)
        v_232 = self.pnnx_unique_74(v_231)
        v_233 = self.model_23_cv3_0_2(v_232)
        v_234 = torch.cat((v_224, v_233), dim=1)
        v_235 = self.model_23_cv2_1_0_conv(v_163)
        v_236 = self.pnnx_unique_75(v_235)
        v_237 = self.model_23_cv2_1_1_conv(v_236)
        v_238 = self.pnnx_unique_76(v_237)
        v_239 = self.model_23_cv2_1_2(v_238)
        v_240 = self.model_23_cv3_1_0_0_conv(v_163)
        v_241 = self.pnnx_unique_77(v_240)
        v_242 = self.model_23_cv3_1_0_1_conv(v_241)
        v_243 = self.pnnx_unique_78(v_242)
        v_244 = self.model_23_cv3_1_1_0_conv(v_243)
        v_245 = self.pnnx_unique_79(v_244)
        v_246 = self.model_23_cv3_1_1_1_conv(v_245)
        v_247 = self.pnnx_unique_80(v_246)
        v_248 = self.model_23_cv3_1_2(v_247)
        v_249 = torch.cat((v_239, v_248), dim=1)
        v_250 = self.model_23_cv2_2_0_conv(v_190)
        v_251 = self.pnnx_unique_81(v_250)
        v_252 = self.model_23_cv2_2_1_conv(v_251)
        v_253 = self.pnnx_unique_82(v_252)
        v_254 = self.model_23_cv2_2_2(v_253)
        v_255 = self.model_23_cv3_2_0_0_conv(v_190)
        v_256 = self.pnnx_unique_83(v_255)
        v_257 = self.model_23_cv3_2_0_1_conv(v_256)
        v_258 = self.pnnx_unique_84(v_257)
        v_259 = self.model_23_cv3_2_1_0_conv(v_258)
        v_260 = self.pnnx_unique_85(v_259)
        v_261 = self.model_23_cv3_2_1_1_conv(v_260)
        v_262 = self.pnnx_unique_86(v_261)
        v_263 = self.model_23_cv3_2_2(v_262)
        v_264 = torch.cat((v_254, v_263), dim=1)
        v_265 = [v_234.size(0), 144, -1]
        v_266 = v_234.view(*v_265)
        v_267 = v_249.view(*v_265)
        v_268 = v_264.view(*v_265)
        v_269 = torch.cat((v_266, v_267, v_268), dim=2)
        v_270, v_271 = torch.split(tensor=v_269, dim=1, split_size_or_sections=(64,80))
        v_272 = [v_270.size(0), 4, 16, v_270.size(2)]
        v_273 = v_270.view(*v_272)
        v_274 = torch.transpose(input=v_273, dim0=2, dim1=1)
        v_275 = F.softmax(input=v_274, dim=1)
        v_276 = self.model_23_dfl_conv(v_275)
        v_277 = [v_270.size(0), 4, v_270.size(2)]
        v_278 = v_276.view(*v_277)
        v_279 = torch.unsqueeze(input=v_192, dim=0)
        v_280, v_281 = torch.chunk(input=v_278, chunks=2, dim=1)
        v_282 = (v_279 - v_280)
        v_283 = (v_279 + v_281)
        v_284 = ((v_282 + v_283) / 2)
        v_285 = (v_283 - v_282)
        v_286 = torch.cat((v_284, v_285), dim=1)
        v_287 = (v_286 * v_191)
        v_288 = F.sigmoid(input=v_271)
        v_289 = torch.cat((v_287, v_288), dim=1)
        v_290 = torch.cat((v_289, v_219), dim=1)
        v_291 = (v_290, v_199, )
        return v_291

请问这个如何修改？

**Comment by yangl720** [comment]
pnnx  yolo11n-seg.torchscript inputshape=[1,3,640,640]  √
pnnx  yolo11n-seg.torchscrip ❌
